Jenkinsfile

```diff
@@ -604,7 +599,7 @@ def Build_CK(Map conf=[:]){
                             stash includes: "perf_onnx_gemm_gfx12.log", name: "perf_log_gfx12"
                         }
                         else if ( arch_type == 6 ){
-                            // run standard tests on gfx908
+                            // run basic tests on gfx908
```

Comment:
Neither *standard* nor *basic* doesn't really describe what's happening? Why not *performance*?

Reply:
well, the original "standard" tests used to include 480 gemms and 100 convolutions. but it turned out to take way too long to run on gfx908 or gfx10/11/12. so on those targets we only run "basic" set of 32 gemms.